.buildkite/premerge.steps.yaml

```diff
@@ -1,25 +1,15 @@
 ---
-ci_version: &ci_version "1.1"
-# This is designed to trap and retry failures because agent lost
-# connection. Agent exits with -1 in this case.
-agent_transients: &agent_transients
-  exit_status: -1
-  limit: 3
-
-# BK system error
-bk_system_error: &bk_system_error
-  exit_status: 255
-  limit: 3
-  
-# Job was interrupted by a signal (e.g. ctrl+c etc)
-bk_interrupted_by_signal: &bk_interrupted_by_signal
-  exit_status: 15
-  limit: 3
-
-# Hook failure
-bk_hook_failure: &bk_hook_failure
-  exit_status: 127
-  limit: 3
+ci_version: &ci_version "${CI_VERSION:-1.1}"
+common_env_vars: &common_env_vars
+  BUILD_TYPE: "GDK"
+  GDK_BRANCH: "${BUILDKITE_BRANCH}"
+  PROJECT_BRANCH: "${PROJECT_BRANCH:-master}"
+  USE_FASTBUILD: "${USE_FASTBUILD:-True}"
+  IS_BUILDKITE_BUILD: "${IS_BUILDKITE_BUILD:-True}"
+  BUILD_ANDROID: "${BUILD_ANDROID:-False}"
```

Comment:
Hmmmm, we are losing the ability to disable certain build aspects on certain Engine versions with this, not sure if I like that

Reply:
For example sometimes we disable Fastbuild during Engine upgrades, or Android might work for a specific version but not another etc

Reply:
Oh, you can just take it out of the anchor then, I think
So if e.g. fastbuild differs, you would delete it from here and add it to each engine thing
That was my intent when I named it the common_vars
If most are the same, I would like to reduce duplication